Jenkinsfile

```diff
@@ -1,6 +1,9 @@
 if (infra.isRunningOnJenkinsInfra()) {
     // ci.jenkins.io
     buildPlugin(platforms: ['linux'])
+
+    // Integration tests
+    essentialsTest(baseDir: "src/test/it")
```

Comment:
I guess this `baseDir` could just be the default?

Reply:
Not sure. In some cases essentials.yml is located in the root (e.g. Core or Git Plugin). CC @raul-arabaolaza . Currently I use root dir as a default

Reply:
I also use root dir as the default (and is the default location used by pipeline-library) but there is no strong technical reason for that, note that in any case any path can be used as is a param of the step itself